nemo_text_processing/inverse_text_normalization/ru/taggers/tokenize_and_classify.py

```diff
@@ -114,7 +114,6 @@ def __init__(self, cache_dir: str = None, overwrite_cache: bool = False):
 
             graph = delete_space + graph + delete_space
             self.fst = graph.optimize()
-            generator_main(far_file, {"tokenize_and_classify": self.fst})
```

Comment:
Intentionally removed?

Reply:
yes this lines is duplicated a few lines below